src/openrct2/ride/Ride.cpp

```diff
@@ -2542,6 +2539,7 @@ void ride_prepare_breakdown(Ride* ride, int32_t breakdownReason)
                         vehicle->update_flags |= VEHICLE_UPDATE_FLAG_BROKEN_CAR;
                 }
             }
+	    ride->inspection_station = 0;
```

Comment:
The indentation on this line (and most others) is not correct. This is also why the CI fails.

As for the line itself: can't this breakdown occur at *any* station? Resetting it to 0 doesn't look quite right.

Reply:
AAGH spaces/tabs [I have emacs set at 8 space tab stops]. Well, that's fixed now.

And this is a breakdown of a vehicle, not of a station, so it doesn't really matter what station.